addOns/webuipoc/src/main/pocs/reactWebUI/src/Components/Sidebar/SidebarData.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import HomeIcon from '@mui/icons-material/Home';
+import SettingsIcon from '@mui/icons-material/Settings';
+import HighlightOffIcon from '@mui/icons-material/HighlightOff';
+
+
+export const SidebarData = [ 
+
+    {
+        title: "Home",
+        icon: <HomeIcon />,
+        link: "/home"
+    } ,
+
+        {
```

Comment:
We should add a formatter as well, to ensure consistent indentation.

Reply:
Won't that be handled by eslint?

Reply:
I didn't check, since eslint was already added and it still had inconsistent indentation I assumed not but if it does.

Reply:
ESLint does handle the formatting as well. ESLint was added but I guess we didn't run it that's why it didnt fix the issues.

Reply:
CI check is already present, I don't think we should rely on pre-commit hook though.